sqlmesh/core/model/definition.py

```diff
@@ -77,8 +77,6 @@
 RUNTIME_RENDERED_MODEL_FIELDS = {
     "audits",
     "signals",
-    "description",
-    "cron",
```

Comment:
These were added some time ago to avoid parsing & rendering things like `cron="@daily"`, or `description="foo ... @bar ..."`, where `@bar` is just a literal substring of the description, not a variable reference.

I'm not sure if this is the right approach, because we may have more fields with "string literals", where we don't care about rendering variables. I decided to simply preserve what we do today to avoid increasing the scope, but this is probably worth discussing & addressing in a followup PR.